server/controllers/user.js

```diff
@@ -91,4 +92,97 @@ export default class Users {
       res.status(400).json({ message: 'invalid email' });
     }
   }
+
+  /**
+   * @description This controller method sends password reset link e-mail
+   *
+   * @param {object} req - Express request object
+   * @param {object} res - Express response object
+   * @returns {object} Json response
+   */
+  static async resetPasswordEmail(req, res) {
+    const { email } = req.body;
+
+    const hashedEmail = HelperUtils.hashPassword(email);
+
+    try {
+      const user = await User.findOne({
+        where: { email }
+      });
+
+      if (user === null) {
+        response(res).notFound({
+          message: 'user not found in our records'
+        });
+      } else {
+        HelperUtils.sendMail(
+          email,
+          'Authors Haven <no-reply@authorshaven.com>',
+          'Password Reset',
+          'Reset Password',
+          passwordResetMarkup(user.firstname, email, hashedEmail)
+        );
+        response(res).success({
+          message: 'Please, verify password reset link in your email box'
+        });     
+      }
+    } catch (err) {
+      response(res).sendData(400, {
+        message: err
+      });
+    }
+  }
+
+  /**
+   * @description This controller method resets user password
+   *
+   * @param {object} req - Express request object
+   * @param {object} res - Express response object
+   * @return {object} Json response
+   */
+  static async resetPassword(req, res) {
+    const { newPassword, confirmPassword } = req.body;
+    const isPassword = newPassword === confirmPassword;
+
+    if (!isPassword) {
+      response(res).sendData(400, {
+        message: 'The supplied passwords do not match'
+      });     
+    }
+
+    try {
+      const hashPassword = HelperUtils.hashPassword(newPassword);
+      
+      const { email, hash } = req.query;
+      const isEmail = await HelperUtils.comparePasswordOrEmail(email, hash);
+      console.log(isEmail);
```

Comment:
Why do you have need of the console.log statement

Reply:
Thank you for pointing this out. I'll fix now and revert.